app/styles/modules/_registration.scss

```diff
@@ -0,0 +1,77 @@
+@import '../variables';
+@import '../mixins';
+
+.registration {
+    margin: 150px auto;
+    width: 80%;
+
+    &>div {
+        margin: 25px;
+        position: relative;
+        xmargin-left: 25px;
+        xmargin-right: 25px;
+    }
+
+    .registration__input {
+        width: 60%;
+        margin: auto;
+        font-size: 1em;
+        line-height: 1.231;
+    }
+
+    input[type=submit] {
```

Comment:
attribute value should be wrapped in quotes - https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors